### ISSUE TITLE
JAVA_OPTS support?

### ISSUE THREAD

**Issue by tsunamaru** [tgbyte/docker-apacheds#1]
Hey,

First of all, nice work!

Can you add JAVA_OPTS environment or something similar to image? Or, if you accept pull requests, I can propose my own maybe?

Thanks!

**Comment by ginkel** [comment]
Have you tried whether setting `JAVA_OPTS` maybe already works out-of-the-box? IMHO the JVM will automatically pick up the env variable.

**Comment by tsunamaru** [comment]
Yup, tried already, seems no difference. I think JVM itself is not picking any environment variables, you have to pass it explicitly in launch string.
